bedrock/mozorg/templates/mozorg/mpl/2.0/includes/structured-data-faq.html

```diff
@@ -0,0 +1,193 @@
+      }
+    }, {
+      "@type": "Question",
+      "name": "I want to use software which is available under the MPL. What do I have to do?",
+      "acceptedAnswer": {
+        "@type": "Answer",
+        "text": "<p>Nothing. Like all other free and open source software, software available under the MPL is available for anyone (including individuals and companies) to use for any purpose. The MPL only creates obligations for you if you want to distribute the software outside your organization.</p>"
+      }
+    }, {
+      "@type": "Question",
+      "name": "I want to distribute software which is available under the MPL, either changed or unchanged, within my organization. What do I have to do?",
+      "acceptedAnswer": {
+        "@type": "Answer",
+        "text": "<p>Nothing. The right to private modification and distribution (and inside a company or organization counts as 'private') is another right guaranteed by free and open source software licenses, including the MPL.</p>"
+      }
+    }, {
+      "@type": "Question",
+      "name": "I want to distribute (outside my organization) complete and unchanged executable programs built from MPL-licensed software by someone other than me. What do I have to do?",
+      "acceptedAnswer": {
+        "@type": "Answer",
+        "text": "<p>As long as the people who distributed the program to you have complied with the MPL, typically nothing. To check and see if the people who distributed the program to you have complied with the MPL, look for the notice that tells you where the software is available in Source Code form (i.e., check that it complies with Section 3.2(a)), and then check that the Source Code is available in that place, including a notice that informs you that the Source Code is available under the terms of the MPL (i.e., check that it complies with Section 3.1).</p>\n\n<p>If you are only distributing libraries, or are only distributing some parts of the program as you received it, it could be that you need to take extra steps to make sure that users of your program are appropriately informed of their rights, as required by section 3.2(a).</p>\n\n<p>In the case of Mozilla Firefox, the Mozilla-provided executable programs already meet the requirements of Section 3, including the notices required by Section 3.1 and 3.2.</p>\n\n<p>If you want to add your own terms when you distribute the software, Section 3.2(b) requires that those terms must not restrict a recipient's rights under the MPL, and if you offer a warranty on the software, Section 3.5 requires you to make clear that it is offered by you alone.</p>"
```

Comment:
Did you mean to include `\n\n` here or maybe a tool did it for you? I think `<p>` is good enough here and in all the other places as well.

Reply:
Yes this was a tool, I will remove it, as I have to close this commit anyways. I dont know how to get the legal doc out 😅